ui/about.go

```diff
@@ -6,21 +6,43 @@ package ui // import "miniflux.app/ui"
 
 import (
 	"net/http"
+	"sort"
 
+	"miniflux.app/config"
 	"miniflux.app/http/request"
 	"miniflux.app/http/response/html"
 	"miniflux.app/ui/session"
 	"miniflux.app/ui/view"
 	"miniflux.app/version"
 )
 
+type option struct {
+	Key string
+	Value string
+}
+
 func (h *handler) showAboutPage(w http.ResponseWriter, r *http.Request) {
 	user, err := h.store.UserByID(request.UserID(r))
 	if err != nil {
 		html.ServerError(w, r, err)
 		return
 	}
 
+	// User must be AdminUsername. To avoid leaking AdminPassword and other secrets to other users.
+	showOptions := user.IsAdmin && (user.Username == config.Opts.AdminUsername())
```

Comment:
`ADMIN_USERNAME` is used only for automatic user creation. Nothing prevent people to remove this variable after the first install. You could also have more than one admin.

Checking if the user is admin should be enough.